Source/Blazorise/wwwroot/blazorise.css

```diff
@@ -1060,6 +1060,10 @@ thead tr th {
   position: relative;
 }
 
+td:focus {
```

Comment:
I would like it more to have any extra CSS for each provider individually instead of in core Blazorise.